### ISSUE TITLE
Lockscreen disables buttons along bottom

### ISSUE THREAD

**Issue by jez9999** [Pi-Developers/Pi-Locker#4]
When I go into the lockscreen, the "square" button on the right disappears.  The second time I do it, the "circle" button in the centre disappears, only to leave the "back triangle" on the left.

This is such a serious bug because it disables core functionality that I have had to turn the lockscreen off until it is fixed.

**Comment by Pi-Developers** [comment]
This is to disable the soft buttons to improve security and should disappear after unlock

**Comment by jez9999** [comment]
No, the buttons are staying removed after unlock.  This is a major bug.

**Comment by Pi-Developers** [comment]
Can you give me a full logcat for your phone please?

**Comment by Lonami** [comment]
For me is kind the opposite. I'm able to skip the lock by pressing the Recent apps soft button, and the back one simply doesn't work.
